From: Juergen Beisert <jbe@pengutronix.de>
Subject: PWM based beeper: adapt it to the new PWM framework and its symbols

Signed-off-by: Juergen Beisert <jbe@pengutronix.de>

---
 drivers/input/misc/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-3.7/drivers/input/misc/Kconfig
===================================================================
--- linux-3.7.orig/drivers/input/misc/Kconfig
+++ linux-3.7/drivers/input/misc/Kconfig
@@ -444,7 +444,7 @@ config INPUT_PCF8574
 
 config INPUT_PWM_BEEPER
 	tristate "PWM beeper support"
-	depends on HAVE_PWM
+	depends on PWM
 	help
 	  Say Y here to get support for PWM based beeper devices.
 
