---
 Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-3.9/Makefile
===================================================================
--- linux-3.9.orig/Makefile
+++ linux-3.9/Makefile
@@ -2,7 +2,7 @@ VERSION = 3
 PATCHLEVEL = 9
 SUBLEVEL = 6
 EXTRAVERSION =
-NAME = Black Squirrel Wakeup Call
+NAME = Unicycling Gorilla
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
