From: Juergen Beisert <jbe@pengutronix.de>
Date: Wed, 28 Jan 2015 19:02:23 +0100
Subject: [PATCH] s3c2410fb: Don't forget to remove the device attribute

Signed-off-by: Juergen Beisert <jbe@pengutronix.de>
---
 drivers/video/fbdev/s3c2410fb.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/video/fbdev/s3c2410fb.c b/drivers/video/fbdev/s3c2410fb.c
index 81af5a63e9e1..77a779b75907 100644
--- a/drivers/video/fbdev/s3c2410fb.c
+++ b/drivers/video/fbdev/s3c2410fb.c
@@ -1049,6 +1049,7 @@ static int s3c2410fb_remove(struct platform_device *pdev)
 	iounmap(info->io);
 
 	release_mem_region(info->mem->start, resource_size(info->mem));
+	device_remove_file(&pdev->dev, &dev_attr_debug);
 
 	framebuffer_release(fbinfo);
 
