---
 Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: barebox-2011.04.0/Makefile
===================================================================
--- barebox-2011.04.0.orig/Makefile
+++ barebox-2011.04.0/Makefile
@@ -1,7 +1,7 @@
 VERSION = 2011
 PATCHLEVEL = 04
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION = -mini2440-master
 NAME = Amissive Actinocutious Kiwi
 
 # *DOCUMENTATION*
